deeprankcore/DataSet.py

```diff
@@ -415,9 +420,8 @@ def create_index_molecules(self):
                     if self.filter(fh5[k]):
                         self.index_complexes += [(fdata, k)]
                 fh5.close()
-            except Exception as inst:
-                print("\t\t--> Ignore File : " + fdata)
-                print(inst)
+            except:
+                _log.exception(f"on {fdata}")
```

Comment:
Prospector fails: pylint: `bare-except / No exception type(s) specified`. Please modify line 423 to `except Exception:`